ridley/ridley.cabal

```diff
@@ -42,7 +42,7 @@ library
                        mtl,
                        shelly,
                        transformers,
-                       prometheus,
+                       prometheus > 0.5.0 && < 2.2.1,
```

Comment:
Why do we need the restrictive upper bound on `prometheus`?

Reply:
We need a lower bound because older versions changed the name of the module hierarchy, whereas newer version (i.e. >= 2.2.1) stopped exporting/defining the `serveHttpTextMetrics` function, which we need.

I guess we could use CPP to get rid of the lower bound and do some copy/paste to get rid of the upper bound, but I have taken the lazy approach for now.

Reply:
Fair enough. It seems to be fine as my build has gone through with the slightly older version.